087-external-certificate-manager.md

```diff
@@ -0,0 +1,241 @@
+* On the next Cluster operator reconciliation Strimzi will copy the new certificate over to `<CLUSTER_NAME>-clients-ca-cert` (renaming and keeping the old certificate), increment the cert generation annotation and add the key generation annotation.
+* Strimzi will roll the Kafka pods to trust the new CA cert.
+* On the next User Operator reconciliation Strimzi will create `Certificate` resources for all the existing `KafkaUser` custom resources.
+
+When using this feature for the ClusterCa:
+* On the next Cluster operator reconciliation Strimzi will copy the new certificate over to `<CLUSTER_NAME>-cluster-ca-cert` (renaming and keeping the old certificate), increment the cert generation annotation and add the key generation annotation.
+* Strimzi will roll the pods once to trust the new CA cert.
+* Strimzi will create `Certificate` resources for all the components and wait for the certificates to be issued.
+* Strimzi will copy over the new certificates into the pod Secrets.
+* Strimzi will roll the pods to use the new certificates.
+
+Once all the pods have the correct cert generation annotation Strimzi can update the `<CLUSTER_NAME>-cluster-ca-cert` and/or `<CLUSTER_NAME>-clients-ca-cert` Secrets to remove the old CA cert and delete the `<CLUSTER_NAME>-cluster-ca` and/or `<CLUSTER_NAME>-clients-ca` Secrets.
+
+### Stopping using this feature
+
+To revert to user managed CAs the user must:
+1. Pause reconciliation for their Kafka cluster.
+2. Update the `<CLUSTER_NAME>-cluster-ca-cert` and/or `<CLUSTER_NAME>-clients-ca-cert` Secrets to:
+   1. contain their public CA cert (keeping the old cert-manager one)
+   2. increment the `strimzi.io/ca-cert-generation` annotation
+3. Create the `<CLUSTER_NAME>-cluster-ca` and/or `<CLUSTER_NAME>-clients-ca` private key Secrets
+4. Update the `Kafka` resource to change the `clusterCa.certificateIssuer` and/or `clientsCa.certificateIssuer` `type` to `internal`.
+5. Resume reconciliation.
+
+When using this feature for the ClientsCa:
+* On the next Cluster Operator reconciliation Strimzi will roll the Kafka pods to trust the new CA cert.
+* On the next User Operator reconciliation Strimzi will issue new certificates for all the existing `KafkaUser` custom resources.
+
+When using this feature for the ClusterCa:
+* On the next reconciliation Strimzi will first roll the pods once to trust the new CA cert.
+* Strimzi will issue new certificates for all the components.
+* Strimzi will roll the pods to use the new certificates.
+
+Once all the pods have been rolled the user can update the `<CLUSTER_NAME>-cluster-ca-cert` and/or `<CLUSTER_NAME>-clients-ca-cert` Secrets to remove the old CA cert.
+The user is responsible for removing the old `Certificate` resources and uninstalling cert-manager.
+
+Notes:
+* Today we do not document how to go from using user managed CAs to Strimzi managed CAs.
+  For this reason I have not included how to go from cert-manager CAs to Strimzi managed CAs.
```

Comment:
Is the note irrelevant because you included it now?

Reply:
I've included how to go from cert-manager CAs to user-managed CAs, but not how to go to Strimzi managed CAs.